docs/messaging/kafka.md

```diff
@@ -159,4 +158,20 @@ This section defines how to apply semantic conventions when collecting Kafka met
 | messaging.kafka.consumer.lag_sum              | Gauge         | Int64      | lag sum | `{lag sum}` | Current approximate sum of consumer group lag across all partitions of topic | `group` | The ID (string) of a consumer group |
 |                                               |               |            |         |             |                                                                              | `topic` | The ID (integer) of a topic |
 
+### Broker Metrics
+
+**Description:** Kafka Broker level metrics.
+
+| Name                                         | Instrument    | Value type | Unit   | Unit ([UCUM](/docs/general/metrics.md#instrument-units)) | Description    | Attribute Key | Attribute Values |
+| ---------------------------------------------| ------------- | ---------- | ------ | -------------------------------------------- | -------------- | ------------- | ---------------- |
+| messaging.kafka.brokers.count                | UpDownCounter | Int64      | brokers   | `{broker}`   | sum of brokers in the cluster | | |
+| messaging.kafka.brokers.consumer.fetch.rate  | Gauge         | Double     | fetches per second | `{fetch}/s`  | Average consumer fetch Rate. | `state` | `in`, `out` |
```

Comment:
If brokers can report `messaging.kafka.brokers.consumer.fetch.count`, it can provide more information and the rate can be derived from it - can it be changed?

Reply:
Indeed. I thought that it was preferred not to have rate metrics but instead use a counter from which the rate can be derived for whatever time period is desired.

Reply:
Hmm, yeah I think that is correct. This should be updated.